--- origsrc/telepathy-sofiasip-0.5.18/src/Makefile.am	2009-08-21 04:15:02.000000000 -0500
+++ src/telepathy-sofiasip-0.5.18/src/Makefile.am	2010-01-25 22:26:17.020325400 -0600
@@ -58,7 +58,7 @@ signals-marshal.h: ${srcdir}/signals-mar
 	--vtail "      { 0, NULL, NULL }\n    };\n    etype = g_@type@_register_static (\"@EnumName@\", values);\n  }\n  return etype;\n}\n" \
 	$< > $@ || rm -f $@
 
-sofiasip.manager: write-mgr-file
+sofiasip.manager: write-mgr-file$(EXEEXT)
 	if ./write-mgr-file > $@.tmp; then \
 		mv -f $@.tmp $@;\
 	else \
